testing/unit/tp/node_account/test_close_masternode.py

```diff
@@ -0,0 +1,177 @@
+"""
+Provide tests for node account handler close masternode implementation
+"""
+import time
+
+import pytest
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.protobuf.processor_pb2 import TpProcessRequest
+from sawtooth_sdk.protobuf.transaction_pb2 import (
+    Transaction,
+    TransactionHeader,
+)
+
+from remme.protos.node_account_pb2 import (
+    NodeAccountInternalTransferPayload, NodeAccountMethod, NodeAccount, EmptyPayload
+)
+from remme.protos.transaction_pb2 import TransactionPayload
+from remme.settings import ZERO_ADDRESS
+from remme.shared.utils import hash512
+from remme.tp.account import AccountHandler
+from remme.tp.node_account import NodeAccountHandler
+from testing.conftest import create_signer
+from testing.mocks.stub import StubContext
+from testing.utils.client import proto_error_msg
+
+from .shared import *
```

Comment:
Never import all functionality, it could mess the namespaces of the module. Import only necessary objects.

Reply:
Done.